src/execution/__tests__/subscribe-test.ts

```diff
@@ -195,6 +195,15 @@ function subscribeWithBadFn(
   );
 }
 
+function subscribeWithBadArgs(
+  args: ExecutionArgs,
+): PromiseOrValue<ExecutionResult | AsyncIterable<unknown>> {
+  return expectEqualPromisesOrValues(
+    subscribe(args),
+    createSourceEventStream(args),
+  );
+}
```

Comment:
Can `subscribeWithBadFn` now use this function?

Reply:
Yes good idea!

Reply:
https://github.com/graphql/graphql-js/pull/3639/commits/03f1a1ed3e546c5d8edbed84e2c41ecfa6a98f64